internal/metrics/federationin/views.go

```diff
@@ -2,11 +2,12 @@ package federationin
 
 import (
```

Comment:
Are we sure this file is imported to run `init`?

Reply:
`./cmd/federationin/main.go` imports `github.com/google/exposure-notifications-server/internal/federationin`.

Anyway, if the code uses uses `stat.Record` on the measures (e.g. `PullInvalidRequest`), it must have imported this module.